cost-analyzer/values.yaml

```diff
@@ -521,6 +521,8 @@ kubecostModel:
   etlDailyStoreDurationDays: 91
```

Comment:
I really should know more about ETL building 🙈 but this daily setting is totally independent of prom, in this case?

Reply:
> but this daily setting is totally independent of prom, in this case?

Mostly, yes. We have a feature of ETL called "cascading ETL" in which, within the hourly ETL window, hourly ETL data is rolled up into daily ETL data. In steady-state, all daily data generally comes from hourly data. @nikovacevic is this a correct representation?